openshift/performance/scripts/run-all-performance-tests.sh

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env bash
+
+declare ns
+
+# shellcheck disable=SC1091
+source "$(dirname "$0")/setup.sh"
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [[ "${ISOLATE_SERVERLESS}" == "true" ]]; then
+  nodes=$(kubectl get nodes -l=node-role.kubernetes.io/worker --no-headers -o custom-columns=":metadata.name")
+  mapfile -t nodes < <(echo "$nodes")
+
+  # disable scheduling on existing worker nodes
```

Comment:
Keep in mind that these nodes won't accept any additional pod once they are marked unschedulable.

Reply:
Yeah that is the idea whatever runs on them I dont want to interfere with.